src/control/PlanarArmController.h

```diff
@@ -85,12 +85,18 @@ class PlanarArmController {
 	 * @param targetVel The target x velocity.
 	 * @return The new command, which is the new joint positions.
 	 */
-	void set_x_vel(robot::types::datatime_t currTime, double targetVel) {
+	void set_x_vel(robot::types::datatime_t currTime, double targetVel,
+				   robot::types::DataPoint<navtypes::Vectord<N>> jointPos) {
```

Comment:
Why do we need it to be a `DataPoint`? Seems like we immediately just call `getData()`, so we might as well make this parameter a `const navtypes::Vectord<N>&`.